scripts/apps/monitoring/views/monitoring-group-header.html

```diff
@@ -24,6 +24,23 @@
         <button ng-if="showRefresh" class="btn btn--primary btn--icon-only-circle btn--small" ng-click="refreshGroup(group)" tooltip="{{:: 'Refresh' | translate}}" tooltip-placement="left">
             <i class="icon-repeat"></i>
         </button>
+        <div class="sortbar" ng-show="customSortOptions">
+            <div class="dropdown dropdown--align-right dropdown--hover" dropdown__toggle>
+                <button id="order_button" class="dropdown__toggle" dropdown__toggle>
+                    <span translate>{{ customSortOptions[customSortOptionActive].label }}</span>
```

Comment:
No need to translate _here_. It's a dynamic value which should come here already translated.

Reply:
Can't we have dynamic values translated with `<div translate>`?